src/Adapter/Product/AdminProductWrapper.php

```diff
@@ -148,6 +148,7 @@ public function processProductAttribute($product, $combinationValues)
             $combinationValues['attribute_ecotax'] = 0;
         } else {
             // Value is displayed tax included but must be saved tax excluded
+            $combinationValues['attribute_ecotax'] = str_replace(',', '.', $combinationValues['attribute_ecotax']);
```

Comment:
I think a better solution would be to cast the attribute ecotax in float before the if. Isn't it ?